handler/src/main/java/io/netty/handler/ssl/OpenSslContext.java

```diff
@@ -45,6 +46,12 @@
 public abstract class OpenSslContext extends SslContext {
 
     private static final InternalLogger logger = InternalLoggerFactory.getInstance(OpenSslContext.class);
+    // To make it easier for users to replace JDK implemention with OpenSsl version we also use
+    // 'jdk.tls.rejectClientInitiatedRenegotiation' to allow disabling client initiated renegotiation.
+    // Java8+ uses this system property as well.
+    // See http://blog.ivanristic.com/2014/03/ssl-tls-improvements-in-java-8.html
+    private static final boolean DEFAULT_REJECT_REMOTE_INITIATED_RENEGOTIATION
```

Comment:
This is not the default value for `rejectRemoteInitiatedRenegotiation` because we do not respect this value for the server-side. How about `JDK_REJECT_CLIENT_INITIATED_RENEGOTIATION`?